spring-cloud-starter-stream-source-sftp/src/main/resources/application.yml

```diff
@@ -0,0 +1,6 @@
+spring:
+  cloud:
+    stream:
+      bindings:
+        output:
+          contentType: application/x-java-serialized-object
```

Comment:
Ye old carriage return needs to be added.

Reply:
PR is pretty out dated, this will be replaced with an env post processor out of core